docs/components/modeler/desktop-modeler/variables.md

```diff
@@ -0,0 +1,21 @@
+---
+id: variables
+title: Inspect process variables
+sidebar_label: Variables
+description: "Learn how to inspect variables in Desktop Modeler."
+---
+
+Track variables defined in the process diagram using the **Variables** view. This shows where a variable was created, which elements write to the variable, and the variable’s scope. Use this view to see which variables are available for use in an element and to troubleshoot problems with variables.
+
+![Variable Tab](./img/variables-tab.png)
+
+The **Elements** list in the **Variables** panel shows the process and all elements that write to variables.
+
+- To see the variables in the scope of an element in the list, select the element, or hold `Ctrl` to select multiple elements.
```

Comment:
How is this supposed to work? I don't see any effect of clicking with control apart from the focus move.

https://github.com/user-attachments/assets/26d9ded8-0225-4b7d-9784-33168588407f

Reply:
It's shift click, you are right 🙈